deps.py

```diff
@@ -415,13 +415,13 @@ def get_foundry_version():
 
 ####################################################################################################
 
-FOUNDRY_VERSION = "2023-11-01"
+FOUNDRY_VERSION = "2023-11-02"
```

Comment:
Also on this, Foundry archive all their nightly releases, only releases on the 2nd day of each month is kept. So probably need to fix to a version that is released on the 2nd day of each month, otherwise download will fail.